### PR TITLE
BioTransformer: Add EDAM identifiers

### PR DESCRIPTION
**Pull request by xtrojak**
Added xref to BioTransformer entry on [bio.tools](https://bio.tools/biotransformer). Galaxy should pull the EDAM operations implicitly from there.

Close https://github.com/RECETOX/galaxytools/issues/380.

### CODE REVIEW

**Comment by xtrojak** [comment]
Requires https://github.com/RECETOX/galaxytools/pull/385 to fix the CI.

**Comment by xtrojak** [comment]
Galaxy tool version was actually already bumped in https://github.com/RECETOX/galaxytools/pull/382/commits/30eb14b1c3f1f774d269dde022bb6963c33be673.

**Comment by hechth** [comment]
@xtrojak oh lol this didn't show up in the review - must have selected only latest commit or something - sorry, my bad. Will undo it.

File: tools/biotransformer/biotransformer.xml [hunk 1]
```diff
@@ -1,8 +1,11 @@
-<tool id="biotransformer" name="BioTransformer" version="@TOOL_VERSION@+galaxy1" profile="21.09">
+<tool id="biotransformer" name="BioTransformer" version="@TOOL_VERSION@+galaxy2" profile="21.09">
```
Comment (hechth) [inline tools/biotransformer/biotransformer.xml:1]:
```suggestion
<tool id="biotransformer" name="BioTransformer" version="@TOOL_VERSION@+galaxy1" profile="21.09">
```

**Pull request merged**